src/components/TileGallery/TileGallery.story.jsx

```diff
@@ -0,0 +1,241 @@
+/* Used dependencies */
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import {
+  CheckmarkFilled16,
+  Rocket32,
+  StarFilled16,
+  Activity32,
+  Sunny32,
+} from '@carbon/icons-react';
+// import { green40 } from '@carbon/colors';
+import { spacing07 } from '@carbon/layout';
+
+import {
+  TileGallery,
+  TileGallerySection,
+  TileGalleryItem,
+  TileGalleryViewSwitcher,
+  StatefulTileGallery,
+  OverflowMenu,
+  OverflowMenuItem,
+} from '../../index';
+import FullWidthWrapper from '../../internal/FullWidthWrapper';
+
+const overflowComponent = (
+  <OverflowMenu onClick={evt => evt.preventDefault()} style={{ height: spacing07 }}>
+    <OverflowMenuItem itemText="Edit" onClick={action('Edit')} />
+    <OverflowMenuItem itemText="Share" onClick={action('Share')} />
+    <OverflowMenuItem itemText="Move" onClick={action('Move')} />
+    <OverflowMenuItem itemText="Delete" onClick={action('Delete')} />
+  </OverflowMenu>
+);
+
+export const content = (
+  <Fragment>
+    <TileGallerySection
+      title={select('Select with/without section', { Title: 'Section 1', None: null }, 'Title')}
+    >
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Rocket32 fill="black" description="Icon" width={50} height={50} />}
+        icon={<CheckmarkFilled16 fill="#40be65" onClick={action('clicked')} />}
```

Comment:
This hex value doesn't appear to be part of the [IBM design language palette](https://www.ibm.com/design/language/elements/color/). 

Instead of a hex, the color tokens can be imported and used from the carbon colors package
```
import { green50 } from '@carbon/colors';
```

Theres a few additional spots where hex codes are hardcoded like this.